src/main/java/com/minecolonies/rankup/modules/databases/DatabaseModule.java

```diff
@@ -0,0 +1,29 @@
+package com.minecolonies.rankup.modules.databases;
+
+import com.minecolonies.rankup.modules.databases.config.DatabaseConfigAdapter;
+import com.minecolonies.rankup.qsml.modulespec.ConfigurableModule;
+import org.spongepowered.api.service.sql.SqlService;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.spongepowered.api.service.sql.SqlService'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AUselessImportCheck)